pkgs/development/tools/ocaml/dune/default.nix

```diff
@@ -1,26 +1,26 @@
-{ stdenv, fetchFromGitHub, ocamlPackages, opaline }:
+{ stdenv, fetchurl, ocamlPackages, opaline }:
 
 stdenv.mkDerivation rec {
   name = "dune-${version}";
-  version = "1.2.1";
-  src = fetchFromGitHub {
-    owner = "ocaml";
-    repo = "dune";
-    rev = "${version}";
-    sha256 = "14m5zg1b6d0y03c5qsmqmwiwz2h0iys02rlgn6kzm519ck679bkq";
+  version = "1.3.0";
+  src = fetchurl {
```

Comment:
Why not fetchFromGitHub?

Reply:
The asset that is prepared by the authors for the release is not exactly a snapshot of the source tree. In particular the `%%VERSION%%` placeholder has been replaced by `1.3.0`.